### ISSUE TITLE
is this a grammatical mistake?

### ISSUE THREAD

**Issue by JOJOkami** [irods/irods#7872]
## Bug Report
There are two syntax errors in this file lib/core/src/packStruct.cpp
![image](https://github.com/irods/irods/assets/33214017/3d4ab3ec-a0be-4501-a659-6ef6db51163d)
**This line should be changed to extendPackedOutput( unpackedOutput, sizeof( rodsLong_t ), outPtr1 );**
![image](https://github.com/irods/irods/assets/33214017/63a9b33b-e770-4a61-a184-0bc71b208643)

### iRODS Version, OS and Version
irods-4.3.2

### What did you try to do?
cross compiling

### Expected behavior
success

**Comment by korydraughn** [comment]
Those references are for the solaris platform which we do not provide packages for. I haven't checked, but you're probably correct about that code not compiling for solaris. If anything, we'd remove all code for that platform since it isn't maintained.

How did you land in that source file? Are you attempting to compile for solaris?

**Comment by JOJOkami** [comment]
I'm doing cross-compilation and I tried using this macro, but I found a problem. Thanks for your reply. I won't be using it anymore.

**Comment by korydraughn** [comment]
Okay. If you're blocked by something and run into problems, feel free to open a new issue.

You can also post questions in the [iRODS Google Group](https://groups.google.com/g/iROD-Chat). Posting there will reach the entire community.

Good luck.
